packages/plugin-express/README.md

```diff
@@ -0,0 +1,6 @@
+# @bugsnag/plugin-express
+
+A [@bugsnag/js](https://github.com/bugsnag/bugsnag-js) plugin for capturing errors in Express (and Connect) web applications.
```

Comment:
Are we planning on adding an example integration at all? It may be helpful to add some more information on how to install the plugin here (or elsewhere) if not.

Reply:
The docs will be worked on in the coming weeks. I think it'd be a good idea to add examples to this repo as a part of that work.